### DISCUSSION TITLE
S.T.A.L.K.E.R.: Clear Sky Running on a MacBook Air M1 on Linux

### DISCUSSION THREAD

**Discussion by m1m1k4tz** [ptitSeb/box64#2097] [Show and tell]
https://www.youtube.com/watch?v=gMwvtHAjePA

**Comment by ptitSeb** [comment]
You can also try with box32 (it's a build option of box64) and regular wine. It might works better.

**Reply by m1m1k4tz** [reply]
I think that’s gonna be the best solution once the steam client works. I’ve also heard rumors that steam is releasing a native arm client using something called proton-arm64ec which I can only assume will be using hangover since it uses that approach (ARM64EC ABI)
